projects/application/karma.conf.js

```diff
@@ -20,6 +20,7 @@ module.exports = function (config) {
       reports: ['html', 'lcovonly', 'text-summary'],
       fixWebpackSourcePaths: true
     },
+    reporters: ['spec', 'kjhtml'],
     reporters: ['progress', 'kjhtml'],
```

Comment:
Now reporters is defined twice. Can we delete this line?

Reply:
Argh! Fixed.